### PR TITLE
AC-6521 Display a list of internal program events via the Eventbrite API

### PR DESCRIPTION
**Pull request by geofrocker**
#### Changes introduced in [AC-6521](https://masschallenge.atlassian.net/browse/AC-6521)
- add migration to add the "register for events" definition
#### Reproduce the current behavior
- On development, Navigate to [link](http://localhost:8181/admin/mc/navtree/3/change/)
- You should see three nav tree items
#### How to test
- checkout this branch and `make migrate`
- Navigate to [link](http://localhost:8181/admin/mc/navtree/3/change/)
- You should see the new `Register for events` nav tree item

### CODE REVIEW

**Comment by VincentHokie** [comment]
I believe there's a missing accelerate PR here, @geofrocker

File: accelerator/sitetree_navigation/sub_navigation/event_subnav_definition.py [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
-    FLUENT_REDIRECT_URL
+    FLUENT_REDIRECT_URL,
+    REGISTER_FOR_EVENTS_URL
 )
 from accelerator_abstract.models import BaseUserRole
 
 FINALIST = BaseUserRole.FINALIST
 ALUMNI = BaseUserRole.ALUM
 MENTOR = BaseUserRole.MENTOR
+STAFF = BaseUserRole.STAFF
```
Comment (mkiterian) [inline accelerator/sitetree_navigation/sub_navigation/event_subnav_definition.py]: this constant isn't used anywhere

**Comment by VincentHokie** [comment]
Sibling PR was merged in after successful QA, merging this in as well since they were dependent on each other.

File: accelerator/migrations/0051_add_register_for_events_to_event_subnav_items.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    create_events_subnav,
+    create_home_subnav,
+    create_judging_subnav,
+    create_resources_subnav,
+    create_startup_dashboard_subnav
+)
+
+
+def create_subnav_trees_and_items(apps, schema_editor):
+    create_directory_subnav()
```
Comment (mkiterian) [inline accelerator/migrations/0051_add_register_for_events_to_event_subnav_items.py]: only the events subnav definition has been updated, should we be calling the create/update functions for the other subnavs here as well?

**Pull request merged**